python/paddle/fluid/backward.py

```diff
@@ -1209,9 +1209,8 @@ def append_backward(loss,
     is_in_control_flow = current_block_idx != 0
 
     # Double grad is not supported in sub-block (control flow)
-    if not is_in_control_flow:
```

Comment:
Why `if not is_in_control_flow` was removed?

Reply:
I will revert this change, and I think  `_appending_grad_times` should be refined as we discussed offline.